web/src/utils/label.ts

```diff
@@ -11,3 +11,10 @@ export const validateK8SName = (label: string) => {
   const quotesCount = (label.match(/"/g) || []).length;
   return (quotesCount == 0 || quotesCount == 2) && k8sName.test(label);
 };
+
+//TODO: remove kindToAbbr after integration of console ResourceIcon in topology library
+const abbrBlacklist = ['ASS'];
+export const kindToAbbr = (kind: string) => {
```

Comment:
isn't it something we can get from the console plugin SDK? I know the console has something similar, we should try to be consistent with them.

Reply:
The function itself is in `console-dynamic-plugin-sdk/src/utils/k8s/k8s-get-resource.ts` but is not exported.

It's related to `bridge/api-discovery-resources` you can retrieve in your `localStorage`, containing a list of kinds with abbreviations. 
If we don't have an alternative we could query these data however it seems there is not the color field on kinds we are looking for.